proto/storage/proto.lock

```diff
@@ -8904,58 +8904,53 @@
               },
               {
                 "id": 4,
-                "name": "license",
-                "type": "License"
-              },
-              {
-                "id": 5,
                 "name": "priority",
                 "type": "int64"
               },
               {
-                "id": 6,
+                "id": 5,
```

Comment:
These changes have not yet been released so it is valid to update the lock file without breaking compatibility.